src/common/utils/fetch-and-deserialize.ts

```diff
@@ -0,0 +1,25 @@
+import { WorkOS } from '../../workos';
+import { List, ListResponse, PaginationOptions } from '../interfaces';
+import { deserializeList } from '../serializers';
+
+function setDefaultOptions(options?: PaginationOptions): PaginationOptions {
+  return {
+    ...options,
+    order: options?.order || 'desc',
+  };
+}
+
+export async function fetchAndDeserialize<T, U>(
```

Comment:
Across the rest of the library we do `export const fetchAndDeserialize = async ()` and `const setDefaultOptions` so probably best to be consistent.